index.html

```diff
@@ -15,17 +15,29 @@
   </head>
   <body>
       <header class="header-title">
-          <div class="navbar">
-              <a><p>James</p></a>
-              <nav class="navbar-list">
-                  <a href="#portfolio" > Portoflio </a>
-                  <a href="#about" > About </a>
-                  <a href="#contact" > Contact </a>
-              </nav>
-              <div class="icon">
-                <i class="fa fa-bars"></i>
-              </div>
-            </div>  
+        <div class="navbar">
+            <a><p>James</p></a>
+            <nav class="navbar-list" >
+                <i class="fa fa-times" id="closeMenu"></i>
```

Comment:
- [ OPTIONAL] As requested by the previous reviewer, Let's try to add more margin or padding-top as in the Figma template

- [OPTTIONAL] How about making your navigation mobile menu page a height of 100vh which will remove this bottom bararrow_down on the mobile size👍🏻 

- [ OPTIONAL] Great job so far :raised_hands: What about preventing the un-background from scrolling when the navbar is open? You can use a CSS property `overflow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇